src/apps/material/helper.ts

```diff
@@ -146,13 +147,101 @@ export const totalBranchesHaveMaterial = (
   }).length;
 };
 
-export const getInfomediaId = (manifestation: Manifestation) => {
-  const access = manifestation?.access || [];
-  return access.reduce<string | null>((acc, curr) => {
-    const { __typename: type } = curr;
-    if (type === "InfomediaService") {
-      return curr.id;
+export const getInfomediaIds = (manifestations: Manifestation[]) => {
+  const infomediaIds = manifestations
+    .map((manifestation) =>
+      manifestation.access.map((currentAccess) => {
+        return currentAccess.__typename === "InfomediaService"
+          ? currentAccess.id
+          : null;
+      })
+    )
+    .flat();
+  return compact(infomediaIds);
+};
+
+export const getAllUniqueMaterialTypes = (manifestations: Manifestation[]) => {
```

Comment:
This is really minor. But I don't think it is necessary to state that the material types the functions deliver are unique. You would normally expect the types to be unique coming from a utility function like this.
So `getMaterialTypes`would be sufficient